cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -186,6 +211,35 @@ public static CruiseControl fromCrd(Kafka kafkaAssembly, KafkaVersion.Lookup ver
         return cruiseControl;
     }
 
+    /**
+     * Estimates broker disk capacity from supplied storage configuration
+     *
+     * @param storage  Storage configuration for Kafka cluster
+     * @return  Broker disk capacity in MB as a String
+     */
+    private static String estimateDiskCapacityPerBrokerInMb(Storage storage) {
+        long capacity = 0;
+        switch (storage.getType()) {
+            case Storage.TYPE_JBOD:
+                capacity = parseMemory(((PersistentClaimStorage) storage).getSize()) / 1000000;
```

Comment:
So what units is this capacity in? Are these in bytes (base 2) or iso (base 10)?

Reply:
Base 10, but I am going to separate this commit into a different PR!

Reply:
Ok but is that the same base CC uses?

Reply:
Yes base 10 for CC as well. Note that unlike the DISK capacity which is given in MB, the NW_IN and NW_OUT values are given in KB, and the CPU as a percentage